### DISCUSSION TITLE
Input Type="Date" with Wire:Model.Lazy

### DISCUSSION THREAD

**Discussion by jjjrmy** [livewire/livewire#3615] [7. Bugs (archived)]
`<input type="date" wire:model.lazy="inputs.date" />`

As soon as you start typing the year part of the date, it will submit.

Expected behavior:
It should not update until you unfocus.

**Comment by joshhanley** [comment]
@jjjrmy so I just found this https://stackoverflow.com/questions/40762549/html5-input-type-date-onchange-event

> But for <input type='date'> fields, the major browser developers trigger onchange as soon as they detect a "valid" date in the field, even if you're still typing.

So that would be why `.lazy` is firing, as it listens for the change event.

Not ideal.

Not sure how we can work around that though.

**Comment by akamasAT** [comment]
this is quite nasty as unexpected. My workaround was the following since .defer was no option:
in my updated method i made a check
`if (Carbon::parse($this->dateField)->diffInYears(now()) >= 1000) return;`
